docs/source/graph-construction/gs-processing/gpartition/ec2-clusters.rst

```diff
@@ -0,0 +1,122 @@
+======================================
+Running partition jobs on EC2 Clusters
+======================================
+
+Once the :ref:`distributed processing setup<gsprocessing_distributed_setup>` is complete,
+you can start the partition jobs. In summary this doc will provide instructions on how to setup the EC2 cluster and
+start GPartition jobs on EC2 cluster.
+
+Create a GraphStorm Cluster
```

Comment:
We did not refer to our current guidance about creating a graphstorm cluster, because we may move them to somewhere else or even remove them. For now we keep it here directly in the tutorial, but that should be a to-do once we decide where to put this part.